src/woo_publications/metadata/organisation_sync.py

```diff
@@ -94,17 +94,38 @@ def update_organisation(file_path: Path):
                 defaults={**fields, "oorsprong": oorsprong},
             )
 
-    to_export = Organisation.objects.exclude(
+    value_list_organisations = Organisation.objects.exclude(
         oorsprong=OrganisationOrigins.municipality_list
-    ).values_list("pk", flat=True)
+    )
 
-    call_command(
-        "dumpdata",
-        "metadata.organisation",
-        format="json",
+    fixutre_data = serializers.serialize(
+        "json",
+        value_list_organisations,
         indent=4,
-        natural_primary=True,
-        pks=",".join([str(pk) for pk in to_export]),
-        output=file_path,
-        stdout=StringIO(),
+        use_natural_primary_keys=True,
+    )
```

Comment:
if you use `serializers.serialize`, can't you make use of the `fields` option? Then the post-processing of json wouldn't even be needed :thinking: